.github/workflows/node.yml

```diff
@@ -0,0 +1,31 @@
+name: node
+
+on: push
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Start postgres
+        run: docker compose up -d --renew-anon-volumes
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: npm
+      - name: install
+        run: npm install
+      - name: build
+        run: npm run build
+      - name: linter
+        run: npm run lint
+        if: ${{ always() }}
+      - name: test
+        run: npm run test
+      - name: integration test
```

Comment:
It would be great to do integration tests over the promotion process, but, as of today, I would remove the integration tests step that will make the job fail.

Reply:
removed for the moment!